src/graphics.c

```diff
@@ -172,3 +188,92 @@ void graphics_char(int x, int y, char ch, struct graphics_color fgcolor,
     return graphics_bitmap(x, y, FONT_WIDTH, FONT_HEIGHT, &fontdata[u],
                            fgcolor, bgcolor);
 }
+
+uint8_t graphics_verify_color_range(uint8_t x) {
```

Comment:
The parameter for this function (and the function it is used in) is a uin8_t, meaning that the cast has already happened. I think we are trying to do too much here -- the parameter type for color functions is already unit8_t, indicating that the values are between 0 and 255. This seems like enough to me.